pipeline/mist.go

```diff
@@ -16,7 +17,6 @@ import (
 )
 
 const MIST_SEGMENTING_SUBDIR = "source"
-const TARGET_SEGMENT_LENGTH_SECS = "5"
```

Comment:
why are we going with 10 seconds as the default now rather than 5?

Reply:
We were doing 10 for Mediaconvert and I think longer segments suits our architecture better on the Livepeer side (fewer requests to transcoders)